src/Mailer/Message.php

```diff
@@ -394,6 +394,19 @@ public function getReplyTo(): array
         return $this->replyTo;
     }
 
+    /**
+     * Add "Reply-To" address.
+     *
+     * @param string|array $email Null to get, String with email,
+     *   Array with email as key, name as value or email as value (without name)
+     * @param string|null $name Name
+     * @return $this
+     */
+    public function addReplyTo($email, ?string $name = null)
```

Comment:
We don't add new methods in patch releases. Other changes seem fine for `master`. So you should make a separate PR targeting `4.next` to add this method after the other changes are merged.